ENVIRONMENT.rst

```diff
@@ -40,9 +42,10 @@ Environment Configuration Settings
 - **SWIFT_PASSWORD**:
 - **SWIFT_AUTH_VERSION**:
 - **SWIFT_ENDPOINT_TYPE**:
+- **WALE_SWIFT_PREFIX**: (optional) the full path to the backup location on the Swift Storage in the format swift://bucket-name/very/long/path. If not specified Spilo will generate it from WAL_SWIFT_BUCKET.
```

Comment:
Isn't `WAL_SWIFT_BUCKET` missing here?

Reply:
Nope, it is [here](https://github.com/zalando/spilo/pull/315/files#diff-787073732367780d2de9c4a21cfbec09L36).

Reply:
Oh.  

OTOH I think the 'Ditto' parts here and there are referring to a previous version, and some other variables came in between, see the link you gave